ui/app/components/modals/customize-gas-modal.js

```diff
@@ -0,0 +1,91 @@
+const Component = require('react').Component
+const h = require('react-hyperscript')
+const inherits = require('util').inherits
+const connect = require('react-redux').connect
+const actions = require('../../actions')
+const GasModalCard = require('../send/gas-modal-card')
+
+function mapStateToProps (state) {
+  return {}
+}
+
+function mapDispatchToProps (dispatch) {
+  return {
+    hideModal: () => dispatch(actions.hideModal()),
+  }
+}
+
+inherits(CustomizeGasModal, Component)
+function CustomizeGasModal () {
+  Component.call(this)
+
+  this.state = {
+    gasPrice: '0.23',
+    gasLimit: '25000',
+  }
+}
+
+module.exports = connect(mapStateToProps, mapDispatchToProps)(CustomizeGasModal)
+
+CustomizeGasModal.prototype.render = function () {
+  const { hideModal } = this.props
+  const { gasPrice, gasLimit } = this.state
+
+  return h('div.send-v2__customize-gas', {}, [
+    h('div', {
+    }, [
+      h('div.send-v2__customize-gas__header', {}, [
+
+        h('div.send-v2__customize-gas__title', 'Customize Gas'),
+
+        h('div.send-v2__customize-gas__close', {
+          onClick: hideModal,
+        }),
+
+      ]),
+
+      h('div.send-v2__customize-gas__body', {}, [
+        
+        h(GasModalCard, {
+          value: gasPrice,
+          min: 0.0,
+          max: 5.0,
+          step: 0.01,
+          onChange: value => this.setState({ gasPrice: value }),
```

Comment:
super minor: could be
`onChange: gasPrice => this.setState({ gasPrice }),`
same on L64

Reply:
change applied in fb35af8b